### PR TITLE
Fix legacy

### PR DESCRIPTION
**Pull request by dmitrylyzo**
Regression https://github.com/jellyfin/jellyfin-web/pull/4834

**Changes**
- Remove `!` from `start_url`.
- Add `AbortController` polyfill.

**Issues**
Fixes https://github.com/jellyfin/jellyfin-web/issues/4940

### CODE REVIEW

File: src/manifest.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "description": "The Free Software Media System",
     "lang": "en-US",
     "short_name": "Jellyfin",
-    "start_url": "index.html#!/home.html",
+    "start_url": "index.html#/home.html",
```
Comment (thornbill) [inline src/manifest.json:6]: Either way this should be updated, but are `#!` routes not working? We _should_ be automatically updating `#!` routes to the `#` equivalent.

Comment (dmitrylyzo) [reply src/manifest.json:6]: It tries to match `!/home.html` with available routes. Maybe initial navigation isn't normalized somewhere?

Comment (thornbill) [reply src/manifest.json:6]: Ok I'll do some additional testing on those paths but that doesn't block this. 👍

**Review by thornbill — APPROVED** [review]

**Comment by rafma0** [comment]
@dmitrylyzo I'm still seeing a back screen on my samsung, can you please test on yours? thank you

**Pull request merged**